views/vm/show.erb

```diff
@@ -20,16 +20,15 @@ viewable_fws =
   left_items: [part("components/vm_state_label", state: @vm[:state], extra_class: "text-md")]
 ) %>
 
-<div class="grid gap-6">
-  <!-- Vm Detail Card -->
-  <%== part(
-    "components/kv_data_card",
-    data: [
+<% data = [
       ["ID", @vm[:id]],
       ["Name", @vm[:name]],
       ["Location", @vm[:location]],
       ["Size", @vm[:size]],
-      ["Storage", (@vm[:storage_size_gib] > 0) ? "#{@vm[:storage_size_gib]} GB" : nil],
+      ["Storage", (@vm[:storage_size_gib] > 0) ? "#{@vm[:storage_size_gib]} GB" : nil]
+    ]
+  data << ["GPU", @vm[:gpu]] if @project.get_ff_gpu_vm
```

Comment:
If the VM has any GPU device, IMHO, we can show the GPU info without checking the feature flag.